pkg/plugin/framework/common/plugin_config.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+func PluginConfigLabelSelector(kind PluginKind, name string) string {
+	return fmt.Sprintf("velero.io/plugin-config=true,%s=%s", name, kind)
```

Comment:
Regarding feature [Changing PVC selected-node](https://velero.io/docs/v1.11/restore-reference/#changing-pvc-selected-node:~:text=the%20second%20%22dev%22.-,Changing%20PVC%20selected%2Dnode,-Velero%20can%20update), `plugin-config` should not set to true as a const.

Reply:
@kaovilai could you please take a look here